src/objects/npe03__Recurring_Donation__c.object

```diff
@@ -35,6 +35,11 @@
                     <default>false</default>
                     <label>Card Expired</label>
                 </value>
+                <value>
```

Comment:
There is an error with the Description/Help Text for the Current and Next Year Value fields. Can you please replace `&quot;` with `&apos;` ?

Reply:
Will change it , @RadSFDeveloper ! In this PR, there is pipeline character in custom field help text for UseFiscalYearForRecurringDonationValue__c, does it needs to be replaced or its fine?

Reply:
That one is fine, thanks for checking @meena-sfdc .